demos/multi_channel/face_detection_demo/README.md

```diff
@@ -29,25 +29,24 @@ multi_channel_face_detection_demo [OPTION]
 Options:
 
     -h                           Print a usage message
+    -i                           Required. An comma separated list of inputs to process. Each input must be a single image, a folder of images or anything that cv::VideoCapture can process.
+    -loop                        Optional. Enable reading the inputs in a loop.
+    -duplicate_num               Optional. Multiply the inputs by the given factor. For example, if only one input is provided, but -ni is set to 2, the demo uses half of images from the input as it was the first input and another half goes as the second input.
     -m "<path>"                  Required. Path to an .xml file with a trained model.
-      -l "<absolute_path>"       Required for CPU custom layers. Absolute path to a shared library with the kernel implementations
+      -l "<absolute_path>"       Required for CPU custom layers. Absolute path to a shared library with the kernels implementations
           Or
-      -c "<absolute_path>"       Required for GPU custom kernels. Absolute path to an .xml file with the kernel descriptions
+      -c "<absolute_path>"       Required for GPU custom kernels. Absolute path to an .xml file with the kernels descriptions
```

Comment:
This is unrelated to inputs, and questionable in its own right. Don't do it here.

Reply:
Reverted